.github/workflows/nox.yml

```diff
@@ -29,27 +29,27 @@ jobs:
       matrix:
         include:
           - session: test
-            python-versions: "3.9, 3.10, 3.11"
+            python-versions: "3.9, 3.10, 3.11, 3.12"
```

Comment:
We also need to update the `test` session in the noxfile and add classifiers to pyproject.toml to declare support for Python 3.12. See https://github.com/ansible-community/antsibull-core/commit/0ad5aed6042784ce7318d0d9184f62259ada8143.